src/shared-components/Tooltip/Tooltip.tsx

```
@@ -0,0 +1,189 @@
+  return (
+    <>
+      <div
+        ref={triggerRef}
+        className={styles.tooltipWrapper}
+        onMouseEnter={showTooltip}
+        onMouseLeave={hideTooltip}
+        onFocus={showTooltip}
+        onBlur={hideTooltip}
+        onKeyDown={handleKeyDown}
+        aria-describedby={isVisible ? tooltipId : undefined}
+      >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)